_src/pattern-library/sass/base/_reset.scss

```diff
@@ -104,6 +104,13 @@ pre {
 // ----------------------------
 .sr-only {
     @extend %a11y-hide;
+    background: black !important; // providing contrast
```

Comment:
Thanks for the context. I'd prefer a few things:

* Can this use our ``black-t`` and ``white-t`` Sass variables instead of the HTML colors?
* I can see this being used elsewhere in the future, can you make a quick Sass extend in our ``_helpers.scss`` partial and then reference it here and in the ``* {}`` nested rule included below?